### PR TITLE
Vet Clinic DataBase (PR)

### PR DESCRIPTION
**Pull request by Doheera-kosi**
# Milestone Objectives reached: 
- Created a database named vet_clinic.
- Created a table named animals.
- Inserted data into the animals table.
- Wrote queries to:
- Find all animals whose name ends in mon.
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either Agumon or Pikachu.
- List names and escape attempts of animals that weigh more than 10.5kg.
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

## Screenshots

vdb1
vdb2
vdb3
vdb4
vdb5
vdb6
vdb7
vdb8

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE public.animals
+(
+    id integer,
+    name "char",
+    date_of_birth date,
+    escape_attempts integer,
+    neutered boolean,
+    weight_kg numeric,
```
Comment (BenMukebo) [inline schema.sql:10]: - [OPTIONAL ] You can also use `decimal` type on the `weight_kg` column, Decimal data type is also accepted 👍🏻

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (0002, 'Gabumon', '11/15/2018', 2, true, 8);
+	
+INSERT INTO public.animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (0003, 'Pikachu', '07/01/2021', 1, false, 15.04);
+	
+INSERT INTO public.animals(
+	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES (0004, 'Devimon', '12/05/2017', 5, true, 11);
```
Comment (BenMukebo) [inline data.sql]: - [ ] Great job so far 👏🏻 However, The date format for the date data type in PostgreSQL is `yyyy-mm-dd`. This is the format used for both storing data and for inserting data. Kindly change this. For more documentation check this [link](https://www.postgresql.org/docs/current/functions-formatting.html). 🤓

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Doheera-kosi,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
- Descriptive Pull Request  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Doheera-kosi) [reply data.sql]: Thank you for the constructive feedback.
Changes made!

Comment (BenMukebo) [reply data.sql]: Great 👍🏻

File: schema.sql (hunk 1 above)
Comment (Doheera-kosi) [reply schema.sql:10]: Thanks!

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**